client/app/lib/onboarding.coffee

```diff
@@ -0,0 +1,111 @@
+# Local class Step
+class Step
+    # Retrieves properties from config Step plain object
+    constructor: (step) ->
+        if step
```

Comment:
The point is not about underscore; the point is about initialize step with the expected type.